<?xml version="1.0"?>
<launch>

    <!-- publishes trace behind marker tf-frame -->
    <node name="$(anon trace_publisher)" pkg="tf_trace_publisher" type="trace_publisher.py" output="screen">
        <param name="tf_parent" value="/world"/>
        <param name="tf_child" value="/ee_link"/>
        <param name="color_R" value="1.0"/>
        <param name="color_G" value="0.0"/>
        <param name="color_B" value="0.0"/>

        <param name="color_A" value="0.7"/>
        <param name="scale" value="0.01"/>
        <param name="length" value="600"/>
        <param name="rate" value="30"/>
        <param name="id" value="1"/>
    </node>

</launch>
